Manifest build fails

The docker manifest creation fails in circleci.

```console
make manifest SUFFIX=
make: git: Command not found
# Manifest for "ulibc"
DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create -a "quay.io/prometheus/busybox:uclibc" \
	"quay.io/prometheus/busybox-linux-amd64:uclibc" \
	"quay.io/prometheus/busybox-linux-armv7:uclibc" \
	"quay.io/prometheus/busybox-linux-armv8:uclibc"
unsupported manifest media type and no default available: application/vnd.docker.distribution.manifest.v1+prettyjws
make: *** [Makefile:37: manifest] Error 1
Exited with code 2
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -9,7 +9,7 @@ jobs:
     steps:
     - checkout
     - setup_remote_docker
-    - run: apk add make bash
+    - run: apk add make bash git
     - run: docker info
     - run: uname -a
       # Register binfmt_misc targets for cross-building
@@ -24,7 +24,7 @@ jobs:
     steps:
     - checkout
     - setup_remote_docker
-    - run: apk add make bash
+    - run: apk add make bash git
     - run: docker info
     - run: docker run --privileged linuxkit/binfmt:v0.6
     - run: make SUFFIX=

--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@ manifest:
 		"$(REPOSITORY)/$(NAME)-linux-amd64:glibc" \
 		"$(REPOSITORY)/$(NAME)-linux-armv7:glibc" \
 		"$(REPOSITORY)/$(NAME)-linux-armv8:glibc" \
-		"$(REPOSITORY)/$(NAME)-linux-ppc64le:glibc" \
+		"$(REPOSITORY)/$(NAME)-linux-ppc64le:glibc"
 	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push "$(REPOSITORY)/$(NAME):glibc"
 
 	# Manifest for "latest"
@@ -53,7 +53,7 @@ manifest:
 		"$(REPOSITORY)/$(NAME)-linux-amd64:latest" \
 		"$(REPOSITORY)/$(NAME)-linux-armv7:latest" \
 		"$(REPOSITORY)/$(NAME)-linux-armv8:latest" \
-		"$(REPOSITORY)/$(NAME)-linux-ppc64le:latest" \
+		"$(REPOSITORY)/$(NAME)-linux-ppc64le:latest"
 	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push "$(REPOSITORY)/$(NAME):latest"
 
 push:

